frontend/src/app.tsx

```diff
@@ -6,13 +6,14 @@ import Sign from 'components/sign/sign'
 import NotFound from 'components/not-found/not-found';
 import Clinics from 'components/clinics/clinics';
 import DoctorsSearch from 'components/doctors-search/doctors-search';
+import AuthorizedRoute from 'components/authorized-route/authorized-route';
 
 const App: React.FC = () => (
   <Switch>
-    <Route path={AppRoute.ADMIN_PAGE} exact component={AdminPage}/>
     <Route path={[AppRoute.SIGN_IN, AppRoute.SIGN_UP]} component={Sign}/>
-    <Route path={AppRoute.CLINICS} component={Clinics} />
-    <Route path={AppRoute.DOCTORS} component={DoctorsSearch}/>
+    <AuthorizedRoute path={AppRoute.CLINICS} component={Clinics} />
+    <AuthorizedRoute path={AppRoute.DOCTORS} component={DoctorsSearch}/>
+    <AuthorizedRoute path={AppRoute.ADMIN_PAGE} exact component={AdminPage}/>
```

Comment:
```suggestion
    <AuthorizedRoute path={AppRoute.Users} exact component={Users}/>
```
Renames everything connected please(folder, file, component and etc)